lib/std/build.zig

```diff
@@ -1841,6 +1842,10 @@ pub const LibExeObjStep = struct {
         self.lib_paths.append(self.builder.dupe(path)) catch unreachable;
     }
 
+    pub fn addSyslibroot(self: *LibExeObjStep, path: []const u8) void {
+        self.syslibroot = path;
+    }
+
```

Comment:
counter proposal: omit this method and add doc comments to the field indicating it is OK for the user to set the field directly.

Reply:
I like that, thanks!